pkg/api/types.go

```diff
@@ -1546,6 +1564,119 @@ func (p *Properties) GetCustomCloudIdentitySystem() string {
 	return AzureADIdentitySystem
 }
 
+// SetCustomCloudProfileEnvironment retrives the endpoints from Azure Stack metadata endpoint and set the values for azure.Environment
```

Comment:
Are the below two functions unit tested?

Reply:
TestSetCustomCloudProfileEnvironmentDefaults or TestSetCustomCloudProfileDefaults